packages/relay-group/src/api/index.ts

```diff
@@ -0,0 +1,66 @@
+// add methods to add relayers to group, remove, pause, resume, etc.
+import { BaseApiClient } from '@openzeppelin/defender-sdk-base-client';
+import {
+  RelayerApiKey,
+  DeleteRelayerApiKeyResponse,
+  CreateKeyParams,
+  RelayerGroupResponse,
+  CreateRelayerGroupRequest,
+  UpdateRelayerGroupRequest,
+} from '../models';
+
+export class RelayGroupClient extends BaseApiClient {
+  protected getPoolId(): string {
+    return process.env.DEFENDER_POOL_ID || 'us-west-2_94f3puJWv';
+  }
+
+  protected getPoolClientId(): string {
+    return process.env.DEFENDER_POOL_CLIENT_ID || '40e58hbc7pktmnp9i26hh5nsav';
+  }
+
+  protected getApiUrl(): string {
+    return process.env.DEFENDER_API_URL || 'https://defender-api.openzeppelin.com/v2/';
```

Comment:
do we still have /v2 in path for SDK?

Reply:
This is api url I think we have not changed it, just FE one.

cc @tirumerla

Reply:
Yep thats right. @MCarlomagno and I will need to sync on this and it should happen in parallel. Will do
it for next release.